### ISSUE TITLE
CO Promotion Package Generation

### ISSUE THREAD

**Issue by eplossl** [TRMN/medusa#194]
With Medusa now holding all the promotion points data (and being the authoritative source for such data) Promotions are now being handled through Medusa.  However, at this time, only promotions through E-6 or O-1/-2 are possible directly by a CO.  All other promotions must be submitted to a promotion review board to be promoted.

Since Medusa is the authoritative source (and will soon have all the data necessary including direct access to awards data) the suggestion is that we make it so that, when a CO chooses to promote a member who is eligible for promotion to something they cannot directly promote to, the system should do the following:

1.  Identify that the CO cannot do the promoting directly (this is already done)
2.  Generate a promotion packet for the member being promoted.  The promotion packet should include, at a minimum:
   a.  The member's name, current rank, RMN number
   b.  Current duty station(s) and any billet(s) assigned
   c.  Training summary (last completed exam for each school attempted, average score)
   d.  Promotion points summary (education points total, award points total, activity points total, perhaps break it out, but that may not be necessary)
3.  (optional) Recommendation form - This would be a form that the user could then fill in with the longer-answer questions associated with a promotion.
4.  (optional) Instruction form - This would be a form for the CO doing the promotion advising how to complete the packet for the board and to whom the paperwork packet (once completed) should be submitted.  This might also be some text that would appear on the screen for the user to read when the packet is ready for download.

Once it is complete (this shouldn't take very long), the system should make this downloadable by either a link or a button on the system.
